src/Cake.Core.Tests/Unit/IO/DirectoryPathTests.cs

```diff
@@ -255,6 +255,25 @@ public void Can_Not_Combine_Directory_Path_With_Absolute_Windows_Directory_Path(
             }
         }
 
+        public sealed class TheGetParentMethod
+        {
+            [WindowsTheory]
+            [InlineData("C:/Data", "C:/")]
+            [InlineData("C:/Data/Work", "C:/Data")]
+            [InlineData("C:/Data/Work/file.txt", "C:/Data/Work")]
+            public void Should_Return_Parent_Directory(string directoryPath, string parentPath)
```

Comment:
We need tests that prove it works with relative paths too

Reply:
I think I've added test support for relative paths. If I haven't, and am being a massive idiot, please advise :)